components/sections/pricing.tsx

```diff
@@ -25,7 +31,11 @@ export const PricingItem = ({ children }) => (
   </li>
 );
 
-export const PricingIncluded = ({ title, children }) => (
+type IncludedProps = {
+  title: string,
+  children: React.ReactNode // TODO: Allow Item type only
```

Comment:
How can I type this `children` prop to restrict the type of the children passed in?
I want it to only allow children with the `Item` component type only.

Reply:
have you tried this one?
```suggestion
  children: React.ReactElement<ItemProps> | Array<React.ReactElement<ItemProps>>
```

if it's work, it should be works also for below one
```suggestion
  children: React.ReactElement<DescriptionProps> | Array<React.ReactElement<DescriptionProps>>
  React.ReactElement<IncludedProps> | Array<React.ReactElement<IncludedProps>>
```

Reply:
Thanks for the suggestion, unfortunately, it doesn't prevent me from rendering another type of component, such as `Pricing.Description`. 😅 
<img width="335" alt="Screenshot 2021-04-10 at 9 46 05 PM" src="https://user-images.githubusercontent.com/6315466/114271878-31deb800-9a46-11eb-9a9f-366f97ba040a.png">